src/ai/.x/templates/openai-chat-quickstart-js/Main.js

```diff
@@ -0,0 +1,36 @@
+const { AzureOpenAI } = require("openai");
+
+// Load the .env file if it exists
+const dotenv = require("dotenv");
+dotenv.config();
```

Comment:
Is this line what puts the env vars into the `process.env` object? Where did the env vars get defined?